README.md

```diff
@@ -126,6 +127,61 @@ const connection = new Connection({
 });
 ```
 
+##### ODBC
+The [ODBC](https://github.com/wankdanker/node-odbc/tree/v2.0) transport establishes a database connection and calls XMLSERVICE stored procedure.
+
+Before using the ODBC transport ensure all dependencies are installed.
+
+Ensure the IBM i Access ODBC driver is installed where you will be running Node.js.
+
+`TODO document installation`
+
+On the client side ensure `unixODBC` and `unixODBC-devel` are both installed.
+
+On IBM i this can be done with:
+
+`yum install unixODBC unixODBC-devel`
+
+Refer to [node-odbc](https://github.com/wankdanker/node-odbc/tree/v2.0#installation) for how this is done your OS.
+
+Also on the client side ensure the `IBM i Access ODBC Driver` is installed.
+
+`TODO document installation`
+
+To use the `ODBC` transport create an instance of Connection with:
+
+```javascript
+const connection = new Connection({
+  transport: 'odbc',
+  transportOptions: { host: 'myhost', username: 'myuser', password: 'mypassword'}
+});
+```
+
+Alternatively you can specify a [DSN](https://support.microsoft.com/en-us/help/966849/what-is-a-dsn-data-source-name) to use.
+
+Create an `.odbc.ini` file within your home directory and add additional configuration options.
+
+Refer to the [knowledge center](https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_73/rzaik/connectkeywords.htm) for a list of valid options.
+
+A simple .odbc.ini file:
+
+```
+[ODBC Data Sources]
+*LOCAL=IBM i Access ODBC Driver
```

Comment:
These 2 lines are not needed with unixODBC driver manager. I think iodbc driver manager needs them, but we don't care about that.